src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -190,10 +195,14 @@ export class SmallArtistHeader extends Component<Props> {
           <Fragment>
             <Carousel
               data={carousel.images as object[]}
-              height={200}
+              height="200px"
```

Comment:
Styled system will infer numerics values, or a string. Why make this change?

Reply:
I was going to only allow for strings so we can pass `"60vh"` so to not get `"60vhpx"` elsehwere.

Reply:
👍 using `HeightProps` in the interface should allow for more flexibility

Reply:
`HeightProps` isn't working so leaving this alone for now.

Reply:
👍